samples/subsys/usb/cdc_acm/sample.yaml

```diff
@@ -6,10 +6,23 @@ tests:
     tags: usb
     harness: console
     harness_config:
-      type: one_line
+      type: multi_line
       regex:
         - "Wait for DTR"
+        - "DTR set, start test"
+        - "Baudrate detected"
+        - "Starting indefinite read/write data loop"
+        - "At least one read/write data cycle completed"
```

Comment:
Have toy tested it working? I have following output:
```
***** Booting Zephyr OS v1.14.0-rc1-447-ga40f5a5c1a *****
Wait for DTR
DTR set, start test
Baudrate detected: 115200
```

Reply:
@nashif For this to work some extra steps needs to be done:
- we need to open another console
- send characters to another console
Do we have command in sanitycheck to open another console and inject characters?